kvbc/test/sparse_merkle_storage/sparse_merkle_db_editor_test.cpp

```diff
@@ -48,6 +51,23 @@ using namespace std::string_literals;
 
 Sliver getSliver(unsigned value) { return toBigEndianStringBuffer(value); }
 
+SetOfKeyValuePairs generateMetadataAndStateTransfer() {
+  constexpr auto num_metadata_object_ids = MAX_METADATA_PARAMS_NUM - INITIALIZED_FLAG;
```

Comment:
Nit: Maybe we can have a note on why this particular number is used. I am not sure I understand it.

Reply:
done